feat(js): Com 145 introduce novunextjs (#6647)
notifirehq/notifire: packages/nextjs/README.md

> # Novu's React SDK for building custom inbox notification experiences.

Reviewer: ```suggestion
# Novu's NextKS SDK for <Inbox />.
```

> Novu provides the `@novu/react` a React library that helps to add a fully functioning Inbox to your web application in minutes. Let's do a quick recap on how you can easily use it in your application.

Reviewer: ```suggestion
Novu provides the `@novu/nextjs` library that helps to add a fully functioning <Inbox /> to your web application in minutes.
```

See full documentation [here](https://docs.novu.co/inbox/react/get-started).

## Installation

> - Install `@novu/react` npm package in your react app

Reviewer: ```suggestion
- Install `@novu/nextjs` npm package in your react app
```

```bash
npm install @novu/nextjs
```

## Getting Started

- Add the below code in the app.tsx file

```jsx
import { Inbox } from '@novu/nextjs';

function Novu() {
  return (
    <Inbox
      options={{
        subscriberId: 'SUBSCRIBER_ID',
        applicationIdentifier: 'APPLICATION_IDENTIFIER',
      }}
    />
  );
}
```

> ## Use your own backend and socket URL

Reviewer: Please move this to the last example. The others are more important.
